service/arm_templates/main.json

```diff
@@ -0,0 +1,152 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterID": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique ID of this guest cluster."
+      }
+    },
+    "storageAccountType": {
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS",
+        "Standard_ZRS",
+        "Premium_LRS"
+      ],
+      "metadata": {
+        "description": "Storage Account type."
+      }
+    },
+    "virtualNetworkCidr": {
+      "type": "string",
+      "defaultValue": "10.0.0.0/16",
+      "metadata": {
+        "description": "The main CIDR block reserved for this virtual network."
+      }
+    },
+    "masterSubnetCidr": {
+      "type": "string",
+      "defaultValue": "10.0.1.0/24",
+      "metadata": {
+        "description": "The masters subnet's CIDR."
+      }
+    },
+    "workerSubnetCidr": {
+      "type": "string",
+      "defaultValue": "10.0.2.0/24",
+      "metadata": {
+        "description": "The masters subnet's CIDR."
+      }
+    },
+    "masterLoadBalancerCidr": {
+      "type": "string",
+      "metadata": {
+        "description": "The masters load balancer's CIDR."
+      }
+    },
+    "workerLoadBalancerCidr": {
```

Comment:
I think this should be `ingressLoadBalancerCidr` as that is what the LB is exposing. This matches what we do in `aws-operator`.